pkg/storage/storage.go

```diff
@@ -103,6 +103,7 @@ type RelationshipTupleReader interface {
 	// ReadPage functions similarly to Read but includes support for pagination. It takes
 	// mandatory pagination options (pageSize can be zero :/)
 	// and returns a slice of tuples along with a continuation token. This token can be used for retrieving subsequent pages of data.
+	// There is NO guarantee on the order returned on the iterator.
```

Comment:
I am not sure if we can reword this a bit - the order will be identical in the same datastore implementation.  (Otherwise, continuation token will not work).  However, there is no guarantee what that order is based on (whether it is based on order of insert, etc.)

Reply:
Updated comment, let me know if it's a bit more clear